node/core/candidate-validation/src/lib.rs

```diff
@@ -490,7 +496,7 @@ where
 }
 
 async fn validate_candidate_exhaustive(
-	mut validation_backend: impl ValidationBackend,
+	mut validation_backend: impl ValidationBackend + Send,
```

Comment:
Suddenly needed this due to the new default implementation of `validate_candidate_with_retry` on the trait. I feel like this bound shouldn't be there, but looks like it gets introduced by the `#[async_trait]` proc macro.

Reply:
Not sure it's helpful here, but there's a `#[async_trait(?Send)]` variant for that attribute.